app/templates/show-project.hbs

```diff
@@ -154,7 +154,7 @@
 
       </div>
       <div class="cell large-5">
-        {{#if model.bbl_featurecollection}}
+        {{#if model.bbls}}
```

Comment:
What is the data type of `bbls`? Any array? Should be fine in the template (which considers empty array falsy).

Reply:
`model.bbls` is an array, yes. On cases when there are no bbls, `model.bbls` just comes back as `null`